force-app/main/default/lwc/rd2EntryFormScheduleSection/rd2EntryFormScheduleSection.js

```diff
@@ -135,6 +124,7 @@ export default class rd2EntryFormScheduleSection extends LightningElement {
                 this.rd2State.dayOfMonth = dd === 31 ? LAST_DAY_OF_MONTH : dd;
             }
             this.cssLastDay = !this.isMonthlyDonation || this.isPaymentModal ? 'slds-hide' : 'slds-p-right_small slds-p-left_small slds-size_12-of-12 slds-large-size_4-of-12 fixExperienceDayOfMonth';
+            this.firstRendered = false;
```

Comment:
`firstRendered` as a var name is not explicitly tied to ExperienceSite users, so should it be set to false regardless of whether `experienceSite` is true to not?

Reply:
Hi @force2b, it should be false regardless of the component being rendered either on the CRM or on the ExperienceSite.
I have moved the variable assignment to outside the condition where it was being changed. Please let me know if you consider further changes are required.